distribution/docs/src/main/resources/_contents/documentationnotescontents.adoc

```diff
@@ -0,0 +1,72 @@
+The {branding} documentation is segmented by user needs, with users categorized as Users, Administrators, Integrators, and Developers.
+
+=== Documentation Outline
+
+The documentation is segmented by user needs, with users categorized as Users, Administrators, Integrators, and Developers. 
+
+Users::
+Users are end users interacting with the applications at the most basic level.
+User help is available through the *Help* link in the User Interface.
+
+Administrators::
+<<Managing {branding}>> |
+Administrators will be installing, maintaining, and supporting existing applications.
+Sections titled with Managing support administrators.
+
+Integrators::
+<<Integrating {branding}>> |
+Integrators will use the existing applications to support their external frameworks.
+Sections titled with Integrating support administrators.
+
+Developers::
+<<Extending {branding}>> |
+Developers will build or extend the functionality of the applications. 
+Sections titled with Extending support administrators.
+
+==== Documentation Updates
+The most current Distributed Data Framework {branding}) documentation is available at: https://codice.atlassian.net/wiki/display/DDF/DDF+Documentation[{branding} Documentation].
+
+==== Conventions
+
+The following conventions are used within this documentation:
+
+===== Callouts
+
+[TIP]
+====
+This is a *Tip*, used to provide helpful information.
```

Comment:
Should we go with IMPORTANT instead of WARNING? Feels like IMPORTANT would apply to more cases than WARNING and get the same level of attention. Just my 0.02 :-)